spec/acceptance/puppetwebhook_spec.rb

```diff
@@ -69,4 +69,14 @@ class { 'puppetwebhook': }
   describe service('puppet_webhook') do
     it { is_expected.to be_running }
   end
+
+  if os[:family] =~ %r{(redhat|suse)}
```

Comment:
The syntax i know to call facts in acceptance is `fact('os.family')`. Why not use it here ?

Reply:
This is pull from the serverspec docs rather than breaker. I have no preference, it was late, I forgot how to check facts in breaker, and serversoec is better documented than breaker (and used by breaker)